### PR TITLE
Added doc strings to base logger file

### PR DESCRIPTION
**Pull request by bamblebam**
## What does this PR do?

Links #9028 
Added doc strings to the base logger file

### Does your PR introduce any breaking changes? If yes, please list them.

None
## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/loggers/base.py [hunk 1]
```diff
@@ -412,15 +415,24 @@ def close(self) -> None:
 
     @property
     def save_dir(self) -> Optional[str]:
+        """
+        Returns `None` as checkpoints should be saved to default / chosen location when using multiple loggers
```
Comment (ethanwharris) [inline pytorch_lightning/loggers/base.py:419]:
```suggestion
        Returns ``None`` as checkpoints should be saved to default / chosen location when using multiple loggers.
```

File: pytorch_lightning/loggers/base.py [hunk 2]
```diff
@@ -380,6 +380,9 @@ def update_agg_funcs(
 
     @property
     def experiment(self) -> List[Any]:
+        """
+        Returns a list of experiment objects for all the loggers in the logger collection
```
Comment (ethanwharris) [inline pytorch_lightning/loggers/base.py:384]:
```suggestion
        Returns a list of experiment objects for all the loggers in the logger collection.
```

File: pytorch_lightning/loggers/base.py [hunk 3]
```diff
@@ -412,15 +415,24 @@ def close(self) -> None:
...
+        """
+        Returns `None` as checkpoints should be saved to default / chosen location when using multiple loggers
+        """
         # Checkpoints should be saved to default / chosen location when using multiple loggers
         return None
 
     @property
     def name(self) -> str:
+        """
+        Returns the experiment names for all the loggers in the logger collection joined by an underscore
```
Comment (ethanwharris) [inline pytorch_lightning/loggers/base.py:427]:
```suggestion
        Returns the experiment names for all the loggers in the logger collection joined by an underscore.
```

File: pytorch_lightning/loggers/base.py [hunk 4]
```diff
@@ -412,15 +415,24 @@ def close(self) -> None:
...
+        """
         return "_".join(str(logger.name) for logger in self._logger_iterable)
 
     @property
     def version(self) -> str:
+        """
+        Returns the experiment versions for all the loggers in the logger collection joined by an underscore
```
Comment (ethanwharris) [inline pytorch_lightning/loggers/base.py:434]:
```suggestion
        Returns the experiment versions for all the loggers in the logger collection joined by an underscore.
```

**Review by ethanwharris — APPROVED** [review]
LGTM, minor formatting comments :smiley:

File: pytorch_lightning/loggers/base.py (hunk 4 above)
Comment (bamblebam) [reply pytorch_lightning/loggers/base.py]: I made all of the changes.

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by Borda — APPROVED** [review]

**Pull request merged**